hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenRejectHandler.java

```diff
@@ -118,9 +142,135 @@ public void pureChecks(@NonNull final TransactionBody txn) throws PreCheckExcept
 
     @Override
     public void handle(@NonNull final HandleContext context) throws HandleException {
+        requireNonNull(context);
         final var op = context.body().tokenRejectOrThrow();
+        final var rejectingAccountID = op.ownerOrElse(context.payer());
         final var rejections = op.rejections();
-        // Todo: Implement the logic for token rejection
+
+        final var hederaConfig = context.configuration().getConfigData(HederaConfig.class);
+        final var ledgerConfig = context.configuration().getConfigData(LedgerConfig.class);
+        validateTrue(rejections.size() < ledgerConfig.tokenRejectsMaxLen(), INVALID_TRANSACTION_BODY);
```

Comment:
```suggestion
        validateTrue(rejections.size() <= ledgerConfig.tokenRejectsMaxLen(), INVALID_TRANSACTION_BODY);
```
Should this be <= ?

Reply:
Ohh! Yes, thats a great catch. 
I've adjusted it.

Thank you 🙏🏻